website/docs/add-secure-apps/flows-stages/stages/authenticator_email/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Email authenticator setup stage
+---
+
+This stage configures an email-based authenticator that sends one-time codes to users' email addresses for authentication.
+
+## Configuration
+
+### Email Settings
+
+The stage can be configured in two ways:
+
+#### Global Settings
+
+Enable the _Use global settings_ option to use authentik's global email configuration. This is recommended if you already have email configured in authentik.
```

Comment:
```suggestion
Enable the **Use global settings** option to use authentik's global email configuration. This is recommended if you already have email configured in authentik.
```

Reply:
"Use global settings" is a toggle in the UI, a boolean variable